### PR TITLE
VACMS-6896 connect with us field

### PR DESCRIPTION
**Pull request by msbtterswrth**
## Description
relates to #6896
~Relates to #2043~ .

## Testing done
Viewed a clp and benefit hub and verified new field is visible on the form and the old fields were hidden. Viewed the section tax and saw that the connect with us fields were hidden. 

## Screenshots
benefit hub:
image
CLP:
image
section: 
image

## QA steps

As cms editor
- [x] 1. Edit a benefits hub page (/node/74/edit) and verify the related office field appears in the right rail section. (nothing was removed on this content type)
- [x] 2. Edit a CLP  (/node/19578/edit) and verify the related office field appears in the connect with us panel, and the 'add a connect with us panel' is removed. 
- [x] 3. View the section taxonomy form display (/admin/structure/taxonomy/manage/administration/overview/form-display) and verify that social media link field, connect with us link field and email updates link text and url have been disabled. 

1. Then validate Acceptance Criteria from issue
- [x] Un-hide the new fields on CLP and Benefit Hub landing page
- [x] Check that all CLP and Benefit Hub landing pages have content as needed
- [x] Hide the old fields on those node types.
- [x] Delete fields from the Section vocabulary, which are no longer in use by the FE (possibly best done as a separate PR)
- [ ] Create follow-up issue to delete the old fields that have been hidden, and work on it.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS team`
- [x] `Sitewide CMS Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by swirtSJW** [comment]
spec tool test failures 
```
      --- Missing fields (present in specification, absent from Drupal)
      | Vocabulary | Sections | Link               | field_link                    | Link                     |  | 1 | Linkit                            |  |
      | Vocabulary | Sections | Link text          | field_email_updates_link_text | Text (plain)             |  | 1 | Textfield                         |  |
      | Vocabulary | Sections | Social media links | field_social_media_links      | Social Media Links Field |  | 1 | List with all available platforms |  |
      | Vocabulary | Sections | URL                | field_email_updates_url       | Text (plain)             |  | 1 | Textfield                         |  |
      +++ Unexpected fields (present in Drupal, absent from specification)
      | Vocabulary | Sections | Link               | field_link                    | Link                     |  | 1 | -- Disabled -- |  |
      | Vocabulary | Sections | Link text          | field_email_updates_link_text | Text (plain)             |  | 1 | -- Disabled -- |  |
      | Vocabulary | Sections | Social media links | field_social_media_links      | Social Media Links Field |  | 1 | -- Disabled -- |  |
      | Vocabulary | Sections | URL                | field_email_updates_url       | Text (plain)             |  | 1 | -- Disabled -- |  | (TravisCarden\BehatTableComparison\UnequalTablesException)
```

And

```
      --- Missing fields (present in specification, absent from Drupal)
      | Content type | Campaign Landing Page | Add a "Connect with us" panel | field_clp_connect_with_us | Entity reference |  | 1 | Select list    |              |
      | Content type | Campaign Landing Page | Related office                | field_related_office      | Entity reference |  | 1 | -- Disabled -- | Translatable |
      +++ Unexpected fields (present in Drupal, absent from specification)
      | Content type | Campaign Landing Page | Add a "Connect with us" panel | field_clp_connect_with_us | Entity reference |  | 1 | -- Disabled -- |              |
      | Content type | Campaign Landing Page | Related office                | field_related_office      | Entity reference |  | 1 | Select list    | Translatable | 
```

and

```
      --- Missing fields (present in specification, absent from Drupal)
      | Content type | Benefits Hub Landing Page | Related office | field_related_office | Entity reference |  | 1 | -- Disabled -- |  |
      +++ Unexpected fields (present in Drupal, absent from specification)
      | Content type | Benefits Hub Landing Page | Related office | field_related_office | Entity reference |  | 1 | Select list |  | 

```

**Comment by msbtterswrth** [comment]
@swirtSJW thanks, i pushed this nearish the end of the day and forgot about it until i saw your comment. just sent a push through that should fix those :)

File: tests/behat/drupal-spec-tool/content_model_vocabulary_fields.feature [hunk 1]
```diff
@@ -28,4 +28,3 @@ Feature: Content model: Vocabulary fields
 | Vocabulary | VHA health service taxonomy | Type of Care | field_vet_center_type_of_care | List (text) |  | 1 | Select list |  |
 | Vocabulary | VHA health service taxonomy | This is a required Vet Center service | field_vet_center_required_servic | Boolean |  | 1 | Single on/off checkbox |  |
 | Vocabulary | Products | Knowledge base landing page | field_kb_landing_page | Entity reference |  | 1 | Autocomplete |  |
-
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/content_model_vocabulary_fields.feature:31]: @msbtterswrth  Please update your config for your IDE.

**Review by swirtSJW — APPROVED** [review]
Looks good.

**Pull request merged**